.gitlab-ci.yml

```diff
@@ -5,6 +5,16 @@
   tags:
     - docker
 
+cppcheck:
+  image: prplfoundationinc/prplmesh-builder:alpine-3.11.3
```

Comment:
Why are we using the apline builder for cppcheck? This should be mentioned in the commit message.

Reply:
Done in a fixup (we use alpine because the version in Ubuntu 18.04 is too old).